lib/resources/r4/encounter.yaml

```diff
@@ -0,0 +1,236 @@
+---
+name: Encounter
+field_name_base_url: http://hl7.org/fhir/R4/encounter-definitions.html#Encounter
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType": "Encounter"
+    }
+  note: resourceType must be Encounter
+  url: http://hl7.org/fhir/R4/json.html#resources
+
+- name: identifier
+  required: 'No'
+  cardinality: 0..*
+  type: Identifier
+  description: The identifiers for the encounter.
+  action: create
+  example: |
+    {
+      "identifier": [
+        {
+          "use": "usual",
+          "type": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/v2/0203",
+                "code": "FIN NBR"
+              }
+            ]
+          },
+          "system": "urn:oid:192.168.0.1",
+          "value": "2009",
+          "period": {
+            "start": "2017-01-17T20:39:40.000Z"
+          }
+        }
+      ]
+    }
+  url: http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.identifier
+
+- name: status
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: The status of the encounter.
+  action: create
+  example: |
+    {
+      "status": "in-progress"
+    }
+  note: Currently do not support `cancelled` or <code>entered-in-error</code>.
+  url: http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.status
+
+- name: class
+  required: 'No'
+  cardinality: 1..1
+  type: code
+  description: The classification of the encounter.
+  action: create
+  example: |
+    {
+      "status": "inpatient"
+    }
+  note: Direct writing of class is not supported. This field will be inferred on a write.
+  url: http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.class
+
+- name: type
+  required: 'Yes'
+  cardinality: 0..*
+  type: CodeableConcept
+  description: The specific type of encounter.
+  action: create
+  example: |
+    {
+      "type": [
+        {
+          "coding": [
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/v2-0004",
+              "code": "O"
+            }
+          ]
+        }
+      ]
+    }
+  url: http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.type
+
+- name: priority
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: The urgency of the of encounter.
+  action: create
+  example: |
+    {
+      "priority": [
+        {
+          "coding": [
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/v3-ActPriority",
+              "code": "O"
+            }
+          ]
+        }
+      ]
+    }
+  url: http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.priority
+
+- name: subject
+  required: 'Yes'
+  cardinality: 1..1
+  type: Reference (Patient)
+  description: The patient present at the encounter.
```

Comment:
Needs to add "action: create"

Reply:
added: https://github.com/cerner/fhir.cerner.com/pull/239/commits/3bbb3b5c2de845dc9a6494d942a5ee43f7ef9198